Utils/annotator/annotator.py

```diff
@@ -0,0 +1,28 @@
+from dataset.dataset import Dataset
+from abc import ABC, abstractmethod
+from tqdm import tqdm
+
+
+class Model(ABC):
+    def __init__(self, model, w2v):
+        self.model = model
+        self.w2v = w2v
+
+    @abstractmethod
+    def __call__(self, s: str) -> float:
+        pass
+
+
+class Annotator:
+    def __init__(self, model: Model):
+        self.model = model
+
+    def annotate(self, dataset: Dataset, window_size: int):
+        data = []
+        dataset.build_crop_sentences(window_size)
+
+        for crop_sentence in tqdm(dataset.crop_sentences):
+            s = [crop_sentence[1] + ' ' + crop_sentence[0].ne + ' ' + crop_sentence[2]]
```

Comment:
Here is better to make next things:
- assing `crop_sentence` for variables;
- use string formatting for the variables: `f'{} {} {}'`